client/gutenberg/extensions/email-subscribe/index.js

```diff
@@ -0,0 +1,37 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
+import EmailSubscribeEdit from './edit';
+import './editor.scss';
+
+export const name = 'email-subscribe';
+
+export const settings = {
+	title: __( 'Email Subscribe', 'jetpack' ),
+	icon: 'email',
+	category: 'widgets',
+	keywords: [ __( 'email', 'jetpack' ), __( 'mailchimp', 'jetpack' ), 'jetpack' ],
+
+	edit: EmailSubscribeEdit,
+
+	// We will generate shortcode as a fallback, so that this works even if gutenberg does not.
+	save: function( props ) {
```

Comment:
Should we move `save` to a separate functional component? It's the common practice around here.

Reply:
I removed whole logic from save, so no need.